cmd/git-s3-push.go

```diff
@@ -3,9 +3,8 @@ package main
 import (
 	"flag"
 	"fmt"
+	s3push "git-s3-push.go"
 	"os"
-
-	"github.com/bobbo/git-s3-push"
```

Comment:
If you rebase onto latest `master` the old import's now working properly (will also fix the build).